README.md

```diff
@@ -1,7 +1,7 @@
 # fiskaltrust interface documentation
-_Welcome to the open source documentation of the fiskaltrust interface!_
+_Welcome to the open-source documentation of the fiskaltrust interface!_
 
-The goal of this repository is to provide a detailed technical specification of the fiskaltrust data interface used by POS-Systems to interact with the fiskaltrust.Middleware, so that the fiskaltrust.Middleware can be integrated by PosCreators into their systems without further support in the most cases.
+This repository aims to provide a detailed technical specification of the fiskaltrust data interface used by POS-Systems to interact with the fiskaltrust.Middleware, so that PosCreators can integrate the fiskaltrust.Middleware into their systems without further support in most cases.
```

Comment:
Minor observation, but it would make more sense possibly if it stated "to provide detailed technical specificationS (plural) of the fiskaltrust etc"

Reply:
to be honest, i don't know... ;-)